ci/Dockerfile-envoy-fuzzit

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 Google Inc.
+# Update 2019 Fuzzit.dev, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+
+RUN apt-get update && apt-get -y install  \
+	build-essential \
+	openjdk-8-jdk   \
+	make            \
+    ninja-build     \
```

Comment:
Nit: indent consistency

Reply:
will remove this file. this is deprecated as there is no need for custom docker file anymore:)